refinery/data_set_manager/tests.py

```diff
@@ -11,49 +11,53 @@
 
 from django.conf import settings
 from django.contrib.auth.models import User
-from django.core.files.uploadedfile import (InMemoryUploadedFile,
-                                            SimpleUploadedFile)
+from django.core.files.uploadedfile import (
+    InMemoryUploadedFile, SimpleUploadedFile
+)
 from django.db.models import Q
 from django.http import QueryDict
 from django.test import LiveServerTestCase, TestCase
 
 from celery.states import PENDING, STARTED, SUCCESS
-from constants import NOT_AVAILABLE
+from constants import NOT_AVAILABLE, REFINERY_SOLR_DOC_LIMIT
```

Comment:
import constants and move it into the "local" import section below since this is not a 3rd party module